baremetal_benchmarking/config_files/sklearnex.manifest.template

```diff
@@ -0,0 +1,66 @@
+# Intel(R) Extension for Scikit-learn* manifest example
+
+loader.entrypoint = "file:{{ gramine.libos }}"
+libos.entrypoint = "{{ entrypoint }}"
+
+loader.log_level = "{{ log_level }}"
+
+loader.env.LD_LIBRARY_PATH = "/lib:{{ arch_libdir }}:/usr/{{ arch_libdir }}:{{ python.stdlib }}/lib:/home/user/.local/lib"
+
+loader.env.HOME = "{{ env.HOME }}"
+
+loader.insecure__use_cmdline_argv = true
+
+loader.env.OMP_NUM_THREADS = "8"
+
+loader.uid = {{ env_user_uid }}
+loader.gid = {{ env_user_gid }}
+
+loader.pal_internal_mem_size = "128M"
```

Comment:
With latest Gramine this manifest option is not required, can be removed.

Reply:
Done.